pallets/token-gateway/src/lib.rs

```diff
@@ -108,17 +114,6 @@ pub mod pallet {
 		/// Account that is authorized to create and update assets.
 		type CreateOrigin: EnsureOrigin<Self::RuntimeOrigin>;
 
-		/// Fungible asset implementation
-		type Assets: fungibles::Mutate<Self::AccountId>
```

Comment:
I would imagine we can add these Traits instead of forking whole token gateway in the final version. @enthusiastmartin might be able to help here